companion to https://github.com/cylc/cylc-flow/pull/4521 (#320)
cylc/cylc-doc: src/user-guide/running-workflows/running-workflows.rst

must be inserted manually at the right cycle point, with ``cylc insert``.

> Files created at workflow start

Reviewer: ```suggestion
Files Created at Workflow Start
```

-------------------------------

Configuration Logs
^^^^^^^^^^^^^^^^^^

> At startup a folder ``log/flow-config`` is created where the flow configuration
> is recorded, with all templating expanded:

Reviewer: ```suggestion
At startup a folder ``log/flow-config`` is created to record the workflow configuration,
with all templating expanded:
```

> - ``flow-processed.cylc`` - A record of the current workflow configuration
>   with templating expanded, but without being fully parsed: Duplicate sections
>   will not be merged.

Reviewer: ```suggestion
- ``flow-processed.cylc`` - the current workflow configuration
  with templating expanded, and duplicate sections unmerged.
```

> - ``<datetime-stamp>-<start/restart/reload>`` - A record of the config at
>   the time a workflow was started, restarted or reloaded, parsed by Cylc:
>   Duplicate sections will be merged.

Reviewer: ```suggestion
- ``<datetime-stamp>-<start/restart/reload>`` - the config at each time the workflow
 was started, restarted, or reloaded; with duplicate sections merged.
```

.. note::

>    These are particularly useful files to look at if you are working with a
>    workflow definition containing many template variables as these are filled
>    in these files.

Reviewer: ```suggestion
   Look at these files to see the result of templating in the workflow config.
```
